website/docs/faqs/Accounts/cloud-upgrade-instructions.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "How to upgrade a dbt Cloud account"
+id: "cloud-upgrade-instructions"
+description: "Instructions for upgrading a dbt Cloud account after the trial ends."
+---
+
+dbt Cloud offers [several plans](https://www.getdbt.com/pricing/) with different features that meet your needs. This document is for dbt Cloud users and explains how to select a plan in order to continue using dbt Cloud. 
+
+:::tip Before you begin
+- You **_must_** be part of the [Owner](/docs/collaborate/manage-access/self-service-permissions) user group to make billing changes. Users not included in this group will not see these options.
+- All amounts shown in dbt Cloud are in U.S. Dollars (USD)   
+:::
+
+
+## Select a plan 
+When your [14 day trial](https://www.getdbt.com/signup/) ends or if your subscription payment is past due , you'll need to select a plan in order to continue using your account:
+
+- Upon logging in, you should see an "Account locked" pop up message with instructions to unlock your account and update your payment details
+- Click **Go to Billing** to go to the billing page
+- Under **Billing**, you can review the available dbt Cloud [plans](https://www.getdbt.com/pricing/) and their features
+
+
+To unlock your account and select a plan, review the following guidance per plan type:
+
+### Developer plan
+
+1. To select a Developer plan, choose the Developer tab under **Billing**. 
+2. Click **Select Plan** on the right. 
+3. Confirm your plan selection on the pop up message. 
+4. This automatically unlocks your dbt Cloud account and you should now enjoy the benefits of the Developer plan. 🎉
+
+<Lightbox src="/img/docs/dbt-cloud/downgrade-dev-flow.gif"/>
+
+### Team plan
+
+1. When your trial expires, your account's default plan enrollment will be a Team plan. 
```

Comment:
If the account is locked, and they have to select a plan, what does it mean that the Team plan is `default`? It might  be good for readers for us to have this above the other instructions if this is the default behavior for all trials.

Reply:
that's right! i initially only left it under the 'team plan' action because i didn't think users that have the intention of downgrading to dev would want to know this but will make that tweak